shell/platform/fuchsia/flutter/BUILD.gn

```diff
@@ -251,6 +251,10 @@ executable("flutter_runner_unittests") {
 
   output_name = "flutter_runner_tests"
 
+  # This is needed by dart/runtime/platform it expects either DEBUG
+  # or NDEBUG to be defined.
+  defines = [ "NDEBUG" ]
```

Comment:
This is not in a public config. So `dart/runtime/platform` wont actually see this. Are you sure this is necessary at all? I thought this was already setup in the toolchain.

Reply:
Interesting. I definitely get an error if I do not include this http://go/paste/6357307015299072